pkg/apis/extensions/types.go

```diff
@@ -638,3 +638,41 @@ type ClusterAutoscalerList struct {
 
 	Items []ClusterAutoscaler `json:"items"`
 }
+
+// PodSelector is a label query over a set of API objects. MatchLabels and
```

Comment:
I suggest just using english rather than type names for this type description: A pod selector is...